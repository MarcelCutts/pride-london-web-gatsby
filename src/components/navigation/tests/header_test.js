import React from 'react'
import { shallow } from 'enzyme'
import Header from '../header'

describe('header', () => {
  it('matches snapshot', () => {
    const wrappedComponent = shallow(<Header />)
    expect(wrappedComponent).toMatchSnapshot()
  })
  it('presents one navitem', () => {
    const wrappedComponent = shallow(<Header />)
    expect(wrappedComponent.find('TestNav')).toHaveLength(1)
  })
})
