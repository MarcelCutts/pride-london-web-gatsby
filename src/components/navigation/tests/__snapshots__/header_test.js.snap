// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header matches snapshot 1`] = `
<Styled(Styled(Box))>
  <Row
    alignItems="center"
    justifyContent="center"
  >
    <TestNav
      items={Object {}}
    />
  </Row>
</Styled(Styled(Box))>
`;
